backup_and_restore/application_backup_and_restore/oadp-performance/oadp-recommended-network-settings.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-recommended-network-settings.adoc"]
+= {oadp-short} recommended network settings
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+:context: oadp-recommended-network-settings
+
+toc::[]
+
+For a supported experience with {oadp-first}, you should have a stable and resilient network across {OCP-short} nodes, S3 storage, and in supported cloud environments.
+
+To ensure successful backup and restore operations, Red Hat recommends that your network settings meet the following minimum requirements:
+
+* Bandwidth: Greater than 2 MB
```

Comment:
Maybe also add something like, `Greater than 2 Mbps for small backups, but consider higher bandwidth (10-100 Mbps) based on the data volume`

Reply:
Incorporated.